### PR TITLE
Ebizzy test for avocado

### PR DESCRIPTION
**Pull request by harish-24**
Ported Ebizzy Test from autotest-client-tests to avocado-misc-tests

### CODE REVIEW

File: cpu/ebizzy.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                   '/ebizzy-0.3.tar.gz')
+        data_dir = os.path.abspath(self.datadir)
+        archive.extract(tarball, self.srcdir)
+        cb_version = os.path.basename(tarball.split('.tar.')[0])
+        self.srcdir = os.path.join(self.srcdir, cb_version)
+
+        patch = self.params.get(
+            'patch', default='Fix-build-issues-with-ebizzy.patch')
+        os.chdir(self.srcdir)
+        p1 = 'patch -p0 < %s/%s' % (data_dir, patch)
```
Comment (apahim) [inline cpu/ebizzy.py:42]: `patch` is failing here.

``` ...
Running 'patch -p0 < /home/apahim/src/avocado-misc-tests/cpu/ebizzy.py.data/Fix-build-issues-with-ebizzy.patch'
[stdout] File configure is read-only; trying to patch anyway
[stdout] patching file configure
[stdout] Hunk #1 FAILED at 31.
[stdout] 1 out of 1 hunk FAILED -- saving rejects to file configure.rej
...
```

Could you please double-check?

File: cpu/ebizzy.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        sm = SoftwareManager()
+        if not sm.check_installed("gcc") and not sm.install("gcc"):
+            self.error("Gcc is needed for the test to be run")
+
+        tarball = self.fetch_asset('http://liquidtelecom.dl.sourceforge.net'
+                                   '/project/ebizzy/ebizzy/0.3'
+                                   '/ebizzy-0.3.tar.gz')
+        data_dir = os.path.abspath(self.datadir)
+        archive.extract(tarball, self.srcdir)
+        cb_version = os.path.basename(tarball.split('.tar.')[0])
```
Comment (apahim) [inline cpu/ebizzy.py]: `cb` is the `compilebench` abbreviation. It has no meaning for this test. Do you mind in rename it?

File: cpu/ebizzy.py (hunk 1 above)
Comment (harish-24) [inline cpu/ebizzy.py:42]: @apahim Some issue with patch file with tabs and spaces. Resolved and sent again. Thank You.

File: cpu/ebizzy.py (hunk 2 above)
Comment (harish-24) [inline cpu/ebizzy.py]: Have renamed it. Thanks.

**Comment by apahim** [comment]
@harish-24 Thank you for the changes. Checking.
For the next PRs, if you don't mind, it would be valuable if you avoid the force update (`git push -f`). For every change in the PR, close the PR in github, create a new local branch out of the first PR branch (i.e. `git checkout -b ebizzy_test_v2` out of `ebizzy_test` branch), amend or add commits and push it sending a new PR from the new branch. This will allow us to see the diff between your changes with `git diff harish-24-ebizzy_test harish-24-ebizzy_test_v2` and also will keep the actual commit in each PR page.

**Comment by apahim** [comment]
Looks good. Thank you!

**Comment by harish-24** [comment]
@apahim  Sure.

**Pull request merged**